src/v2/compiled-lang/en-us.json

```diff
@@ -0,0 +1,98 @@
+{
+  "navbar.artists": [
```

Comment:
whats the diff of the `lang` and `compiled-lang` files? do we need both?

Reply:
The ones in `lang` are the ones added by us.
In case we also adopt a third-party translation tool, these are the ones that are synced between our project and the tool.

The ones in `compiled-lang` are the ones that are actually consumed by the application.

More on this can be found here:
- [Message extraction](https://formatjs.io/docs/getting-started/message-extraction)
- [Message distribution](https://formatjs.io/docs/getting-started/message-distribution)